drivers/rn2xx3/Kconfig

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+source "$(RIOTBASE)/Kconfig.macros"
```

Comment:
If we keep the `Kconfig.macros` file, this should be `source`d in the root Kconfig

Reply:
Done. Updated [b63bc8d ](https://github.com/RIOT-OS/RIOT/pull/13952/commits/b63bc8d4dda1c2d30fab03bdbcab147b43e7318a)